package.json

```diff
@@ -61,8 +61,7 @@
       "src/**/*.{js,jsx,ts,tsx}",
       "!src/**/*.{.test.*}",
       "!src/**/test/**/*",
-      "!src/**/assets/**",
-      "!src/config/**/*"
+      "!src/**/assets/**"
```

Comment:
not to be done now, but should we change the version to v3.0.0?

Reply:
Something to consider. We should talk about this when preparing release